doc/nyan/api_reference/reference_aux.md

```diff
@@ -1220,7 +1224,7 @@ The maximum contingent size.
 ```python
 ResourceAmount(Entity):
     type   : Resource
-    amount : int
+    amount : float
```

Comment:
Hmmm... when would this be useful? We can the resources as a double internally, but for the API having non-fractional resource amounts is much cleaner I think.

Reply:
Mostly when the `ResourceAmount` is not used as an amount but as a multiplier (eg. repair cost per hp) or a rate (eg. villager gather rate). Also for resources introduced by mod, you never know...

Reply:
For rates we have `ResourceRate`.

I think the reason for keeping `ResourceRate` and `ResourceAmount` separate was that `ResouceRate` is tied to a time unit (Resource/time unit) and thus a bit more complex. `ResourceAmount` is supposed to be the less calculation intensive object.